testbed/testbed/mock_backend_test.go

```diff
@@ -23,16 +23,16 @@ import (
 )
 
 func TestGeneratorAndBackend(t *testing.T) {
-	mb := NewMockBackend("mockbackend.log")
+	mb := NewMockBackend("mockbackend.log", NewJaegerReceiver(DefaultJaegerPort))
```

Comment:
Yep, changing to dynamic port. Note: some tests will continue using the default port for now because they use hard-coded ports in collector config file. These can be refactored later.